deltachat-ffi/deltachat.h

```diff
@@ -23,7 +23,7 @@ typedef struct _dc_provider dc_provider_t;
 typedef struct _dc_event    dc_event_t;
 typedef struct _dc_event_emitter dc_event_emitter_t;
 typedef struct _dc_accounts_event_emitter dc_accounts_event_emitter_t;
-
+typedef struct _dc_json_api_instance dc_json_api_instance_t;
```

Comment:
For consistency, I'd propose to either stick to "JSON RPC" or "JSON API" in naming, but not a mixture of both. I think JSON RPC makes more sense, as that's what it is. In that spirit, I'd propose to rename the FFI methods. I'll include suggestions in the review comments.